packages/core/src/additional-components/Background/Background.vue

```diff
@@ -1,9 +1,10 @@
 <script lang="ts" setup>
-import { useVueFlow } from '@vue-flow/core'
 import { computed, toRef } from 'vue'
-import { BackgroundVariant } from './types'
-import type { BackgroundProps } from './types'
+
 import { DefaultBgColors, DotPattern, LinePattern } from './patterns'
+import { type BackgroundProps, BackgroundVariant } from './types'
+
+import { useVueFlow } from '~/composables'
```

Comment:
There's a problem with using tsconfig shortcut paths inside vue component, as they're not resolved to the actual paths when the build is triggered.

This works in `.ts` files thanks to `ts-transform-paths` but, alas, not in `.vue` files -.-

So this sadly has to be `../../../composables/useVueFlow`